ui-tests/cypress/support/pageObjects/users/roles/RolesPage.js

```diff
@@ -76,12 +76,12 @@ class RolesKebabMenu extends KebabMenu {
 
   getDetails() {
     return this.get()
-               .find(`[${DATA_ID}=detail-${this.code}]`);
+               .find(`${htmlElements.li}.RoleListMenuAction__menu-item-detail`);
```

Comment:
How can you be sure it will get the correct _details_ element without the code of the row?

Reply:
I retrieved the cypress documentation where they clarify how functions like .click() and .type() work. 
The action in the Dom generates some sort of context where the visibility of the element is checked by cypress before performing any actions. 

Also, I did some checks to exclude a random match of the randomly generated element in the test. 

Therefore it seems to me that the modification does not affect the purposes of the test.